tools/msconvert/msconvert_macros.xml

```diff
@@ -219,6 +219,13 @@
       fi
       #if $general_options.multi_run_output.do_multi_run_output == 'false':
         && mv 'outputs/${os.path.splitext($basename)[0]}.${output_type}' '${output}'
+      #else
+        ## make mzML and mzXML extensions lower case (the Galaxy data type is ) otherwise detetion of the file
+        #if $output_type == 'mzML' or $output_type == 'mzML'
```

Comment:
could that work `for i in $( ls | grep [A-Z] ); do mv -i "$i" "`echo $i | tr 'A-Z' 'a-z'`"; done`

Reply:
mzML is here twice, I guess the second should be mzXML

Reply:
> could that work `for i in $( ls | grep [A-Z] ); do mv -i "$i" "`echo $i | tr 'A-Z' 'a-z'`"; done`

This would also make the non-extension part of the file name lower case.

Reply:
> mzML is here twice, I guess the second should be mzXML

Should be fixed